tests/e2e/data.js

```diff
@@ -0,0 +1,149 @@
+  let data = null
+  let dummyStringFieldValue = getRandomString()
+  const testClassName = getRandomString()
+  const testSocketName = getRandomString()
+  const instanceName = getRandomString()
+
+  before(function (done) {
+    createTestInstance(instanceName)
+      .then(instanceObj => {
+        process.env.SYNCANO_INSTANCE_NAME = instanceObj.name
+        process.env.SYNCANO_API_KEY = process.env.E2E_ACCOUNT_KEY
+        return new Server()._class.create({
+          name: testClassName,
+          schema: [
+            {type: 'string', name: 'field_string', filter_index: true, order_index: true},
+            {type: 'text', name: 'field_text'},
+            {type: 'integer', name: 'field_integer'},
+            {type: 'float', name: 'field_float'},
+            {type: 'array', name: 'field_array'},
+            {type: 'file', name: 'field_file'}
+          ]
+        })
+      })
+      .then(classObj => {
+        data = new Server().data
+        done()
+      })
+      .catch(err => {
+        console.log(err)
+        err.response.text()
+          .then(text => {
+            console.log(text)
+            done(err)
+          })
+      })
+  })
+
+  after(function (done) {
+    deleteTestInstance(instanceName)
+      .then(() => {
+        done()
+      })
+      .catch(() => {
+        done()
+      })
+  })
+
+  it('can be created', function (done) {
+    data[testClassName].create({field_string: dummyStringFieldValue})
+      .then(dataObj => {
+        expect(dataObj.field_string).to.be.equal(dummyStringFieldValue)
+        expect(dataObj.revision).to.be.equal(1)
+        expect(dataObj).to.not.be.empty
+        done()
+      })
+      .catch(err => {
+        console.log(err)
+        done(err)
+      })
+  })
+
+  it('can be listed', function (done) {
+    data[testClassName]
+      .where('field_string', dummyStringFieldValue)
+      .first()
+      .then(dataObj => {
+        expect(dataObj).to.not.be.empty
+        done()
+      })
+      .catch(err => {
+        done(err)
+      })
+  })
+
+  it('can be deleted', function (done) {
+    data[testClassName]
+      .where('field_string', dummyStringFieldValue)
+      .first()
+      .then(dataObj => {
+        expect(dataObj).to.not.be.empty
+        return data[testClassName].delete(dataObj.id)
+      })
+      .then(() => {
+        return data[testClassName]
+          .where('field_string', dummyStringFieldValue)
+          .first()
+      })
+      .then((resp) => {
+        expect(resp).to.be.null
+        done()
+      })
+      .catch(err => {
+        done(err)
+      })
+  })
+
+  it('can be sorted', function (done) {
+
+    let firstObject = null;
+    let secondObject = null;
+
+    // Create objects
+    Promise.all([
+      data[testClassName].create({field_string: 'abcdef'}),
+      data[testClassName].create({field_string: 'cdefgh'}),
+      data[testClassName].create({field_string: 'bcdefg'})
+    ])
+    .then(objects => {
+      firstObject = objects[0]
```

Comment:
`const [firstObject, secondObject] = objects`

Looks better but I don't know if we have this configured in babel